generate.pl

```diff
@@ -330,15 +330,15 @@ =head1 DESCRIPTION
     && archFlag="$([ "$archBits" = '64' ] && echo '-Duse64bitall' || echo '-Duse64bitint')" \
     && ./Configure -Darchname="$gnuArch" "$archFlag" {{args}} {{extra_flags}} -des \
     && make -j$(nproc) \
-    && {{test}} \
+    && { [ "$(dpkg --print-architecture)" = 'riscv64' ] || {{test}}; } \
```

Comment:
I guess we can now disregard this per https://github.com/Perl/docker-perl/issues/195#issuecomment-3287007418